### PR TITLE
RX logging changes

### PR DESCRIPTION
**Pull request by aherzberg**
## Summary
Original RX logging was not showing any errors when we'd like it to. Trying a new way of logging in staging to see where it shows up. 

## Related issue(s)
https://github.com/department-of-veterans-affairs/va-mobile-app/issues/4923

## Testing done

- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: lib/rx/middleware/response/rx_parser.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class RxParser < Faraday::Response::Middleware
         # @return [Faraday::Env]
         #
         def on_complete(env)
-          Rails.logger.info('RX Parser Debugging', body: env.body, status: env.status)
+          Rails.logger.info('RX Parser Debugging', status: env.status, body: env.body)
```
Comment (aherzberg) [inline lib/rx/middleware/response/rx_parser.rb:16]: in some logs, the character limit is reached and not showing the status. Switching the two so we get it first.

File: lib/rx/middleware/response/rx_failed_station.rb [hunk 2]
```diff
@@ -14,7 +14,6 @@ class RxFailedStation < Faraday::Response::Middleware
         # @return [Faraday::Env]
         #
         def on_complete(env)
-          Rails.logger.info('RX Failed Station Debugging', body: env.body, status: env.status)
```
Comment (aherzberg) [inline lib/rx/middleware/response/rx_failed_station.rb:17]: redundant with the rx_parser

File: lib/rx/configuration.rb [hunk 3]
```diff
@@ -59,8 +59,8 @@ def connection
         conn.request :json
 
         # Uncomment this if you want curl command equivalent or response output to log
-        # conn.request(:curl, ::Logger.new(STDOUT), :warn) unless Rails.env.production?
-        # conn.response(:logger, ::Logger.new(STDOUT), bodies: true) unless Rails.env.production?
+        conn.request(:curl, ::Logger.new($stdout), :warn) unless Rails.env.production?
+        conn.response(:logger, ::Logger.new($stdout), bodies: true) unless Rails.env.production?
```
Comment (jperk51) [inline lib/rx/configuration.rb:63]: Change this env to staging to test before we try prod

**Review by jperk51 — COMMENTED** [review]
Change the env

File: lib/rx/configuration.rb (hunk 3 above)
Comment (aherzberg) [reply lib/rx/configuration.rb:63]: that uses an `unless` so should log in any env that isn't production.

Comment (jperk51) [reply lib/rx/configuration.rb:63]: lol whoops

**Review by jperk51 — APPROVED** [review]
LGTM

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**